images/linux/scripts/installers/rust.sh

```diff
@@ -32,6 +32,10 @@ for cmd in rustup rustc rustdoc cargo rustfmt cargo-clippy bindgen cbindgen; do
     fi
 done
 
+# Cleanup Cargo cache
+rm -rf "${CARGO_HOME}/registry/index/*"
+rm -rf "${CARGO_HOME}/registry/src/*"
```

Comment:
I think this should remove the entire `${CARGO_HOME}/registry`.
`${CARGO_HOME}/registry/cache` will only contain crate archives that happened to be downloaded to build and install `bindgen`/`cbindgen`. These can be only occasionally useful in crate builds in a workflow, which may well use different versions of the same crates depending on what their `Cargo.lock` has settled on.
I think it's better to clean these up entirely from the virtual environment and direct the users to [caching schemes](https://github.com/actions/cache/blob/master/examples.md#rust---cargo).

Reply:
Thanks!